src/ontology/mondo-edit.obo

```diff
@@ -361087,32 +361054,52 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/537/acute-no
 [Term]
 id: MONDO:0018875
 name: Li-Fraumeni syndrome
-def: "Li-Fraumeni syndrome (LFS) is a rare cancer predisposition syndrome characterized by the early-onset of multiple primary cancers such as breast cancer, soft tissue and bone sarcomas, brain tumors and adrenal cortical carcinoma (ACC)." [Orphanet:524]
+def: "A rare cancer predisposition syndrome characterized by the early-onset of multiple primary cancers such as breast cancer, soft tissue and bone sarcomas, brain tumors and adrenal cortical carcinoma (ACC)." [Orphanet:524]
+subset: clingen
 subset: ordo_disease {source="Orphanet:524"}
+synonym: "LFS" RELATED ABBREVIATION [OMIM:151623]
+synonym: "LFS1" RELATED ABBREVIATION [MONDO:Lexical, OMIM:151623]
+synonym: "LFS3" EXACT ABBREVIATION [MONDO:0012234, MONDO:Lexical, OMIM:609266]
 synonym: "Li Fraumeni syndrome" RELATED [GARD:0006902]
 synonym: "Li-Fraumeni familial cancer susceptibility syndrome" EXACT [NCIT:C3476]
 synonym: "Li-Fraumeni familiar cancer susceptibility syndrome" EXACT [DOID:3012, NCIT:C3476]
 synonym: "Li-Fraumeni syndrome" EXACT CLINGEN_PREFERRED [NCIT:C3476]
+synonym: "Li-Fraumeni syndrome" RELATED [OMIM:151623]
+synonym: "Li-Fraumeni syndrome caused by mutation in TP53" EXACT [MONDO:design_pattern]
+synonym: "Li-Fraumeni-like syndrome" RELATED [OMIM:151623]
+synonym: "sarcoma family syndrome of 51 and Fraumeni" RELATED [OMIM:151623]
 synonym: "sarcoma family syndrome of Li and Fraumeni" RELATED [GARD:0006902]
 synonym: "sarcoma, breast, leukaemia and adrenal gland syndrome" EXACT [DOID:3012]
 synonym: "SBLA syndrome" EXACT [DOID:3012]
+synonym: "Sbla syndrome" RELATED [OMIM:151623]
 synonym: "SBLA syndrome (sarcoma, breast, leukemia, and adrenal gland)" RELATED [GARD:0006902]
+synonym: "TP53 Li-Fraumeni syndrome" EXACT [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
+xref: DOID:0111503
 xref: DOID:3012 {source="MONDO:equivalentTo"}
 xref: GARD:0006902 {source="MONDO:equivalentTo"}
 xref: ICD10CM:D48.9 {source="Orphanet:524", source="Orphanet:524/attributed", source="Orphanet:524/ntbt"}
 xref: ICD9:V84.01 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: MedDRA:10066795 {source="Orphanet:524", source="Orphanet:524/e"}
+xref: MESH:C563754
 xref: MESH:D016864 {source="DOID:3012", source="Orphanet:524", source="MONDO:equivalentTo", source="Orphanet:524/e"}
 xref: NCIT:C3476 {source="DOID:3012", source="MONDO:equivalentTo", source="MONDO:exact-label-match"}
-xref: OMIMPS:151623 {source="MONDO:cjm", source="MONDO:equivalentTo"}
+xref: OMIM:151623 {source="MONDO:equivalentTo"}
+xref: OMIM:609266 {source="MONDO:equivalentObsolete"}
+xref: OMIMPS:151623 {source="MONDO:cjm"}
```

Comment:
I removed the MONDO:equivlalentTo here
[
](https://omim.org/phenotypicSeries/PS151623)

Reply:
I agree, the OMIMPS will probably be removed eventually. 
We probably also add this to the excluded list otherwise it will come back during the next slurp

Reply:
oh yes, good point. I'm confused about the excluded list - do we add this to the spreadsheet only?

Reply:
I created a ticket to do so
https://github.com/monarch-initiative/mondo/issues/6485